diff --git a/node_modules/react-native-simple-bottom-sheet/index.js b/node_modules/react-native-simple-bottom-sheet/index.js
index 56d6ef8..97ffd9a 100644
--- a/node_modules/react-native-simple-bottom-sheet/index.js
+++ b/node_modules/react-native-simple-bottom-sheet/index.js
@@ -58,11 +58,11 @@ class BottomSheet extends Component {
   };

   componentDidMount() {
-    BackHandler.addEventListener('hardwareBackPress', this._onBackPress);
+    // BackHandler.addEventListener('hardwareBackPress', this._onBackPress);
   }

   componentWillUnmount() {
-    BackHandler.removeEventListener('hardwareBackPress', this._onBackPress);
+    // BackHandler.removeEventListener('hardwareBackPress', this._onBackPress);
   }

   _setPanResponders() {
