diff --git a/node_modules/react-native-simple-bottom-sheet/index.js b/node_modules/react-native-simple-bottom-sheet/index.js
index 56d6ef8..784be93 100644
--- a/node_modules/react-native-simple-bottom-sheet/index.js
+++ b/node_modules/react-native-simple-bottom-sheet/index.js
@@ -57,13 +57,13 @@ class BottomSheet extends Component {
     return this.state.isPanelOpened;
   };

-  componentDidMount() {
-    BackHandler.addEventListener('hardwareBackPress', this._onBackPress);
-  }
+  // componentDidMount() {
+  //   BackHandler.addEventListener('hardwareBackPress', this._onBackPress);
+  // }

-  componentWillUnmount() {
-    BackHandler.removeEventListener('hardwareBackPress', this._onBackPress);
-  }
+  // componentWillUnmount() {
+  //   BackHandler.removeEventListener('hardwareBackPress', this._onBackPress);
+  // }

   _setPanResponders() {
     this._parentPanResponder = PanResponder.create({
